Code/Mantid/docs/source/algorithms/LoadNMoldyn4Ascii-v1.rst

```diff
@@ -0,0 +1,48 @@
+.. algorithm::
+
+.. summary::
+
+.. alias::
+
+.. properties::
+
+Description
+-----------
+
+Loads data from version 4 of nMOLDYN saved in the ASCII format after being
+extracted from the ``.tar`` archive.
+
+Functions can be provided with ot without the comma between multiple dependant
+variables, for example a function names ``f(q,t_H)`` in nMOLDYN can be loaded
+using either ``f(q,t_H)`` or ``f(qt)_H`` as a function name in this algorithm.
```

Comment:
Is this a typo? `f(qt)_H` -> `f(qt_H)`?

This is working well for me and it comes with tests, documentation, etc. So I think it's ready to be merged.
@DanNixon: Let me know if you'd like to fix the typo, I think we don't need to wait for another build if you edit this.